kernel/src/common/utils.rs

```diff
@@ -60,24 +60,3 @@ macro_rules! storage_volume {
         pub static $N: [u8; $kB * 1024] = [0x00; $kB * 1024];
     };
 }
-
-/// Create an object with the given capability.
-///
-/// ```ignore
-/// use kernel::capabilities::ProcessManagementCapability;
-/// use kernel;
-///
-/// let process_mgmt_cap = create_capability!(ProcessManagementCapability);
-/// ```
-///
-/// This helper macro can only be called in an `unsafe` block, and is used by
```

Comment:
This seems to be incorrect as an `unsafe impl` does not need an `unsafe` block and even `#![forbid(unsafe_code)]` does not prevent this macro from being called in e.g. a capsule. Adding `#[allow(unsafe_code)]` inside of the body can be used to change this behavior tough.